version.py

```diff
@@ -1,50 +1,65 @@
 # Copyright (c) 2019 Aiven, Helsinki, Finland. https://aiven.io/
```

Comment:
I can guess the reason for some of the changes (imp deprecated, etc.), but I'm not sure about the reason behind others ?

Reply:
In the specific file you mean? The main change is going from the ``imp`` module to ``importlib``.
I fixed pylint errors and added print statements for traceability (where do we get the version string from?).
The type hints and variable renaming were not necessary.